server/database/seed.js

```diff
@@ -37,13 +52,12 @@ const seedRules = () => {
     const importantNotesThree = faker.lorem.paragraphs(2);
     const importantNotesFour = faker.lorem.paragraphs(5);
     const importantNotesFive = faker.lorem.paragraphs(1);
-    db.query('INSERT INTO rules (check_in_start, check_in_end, check_out, kid_friendly, credit_cards, age_restriction, curfew, lock_out, non_smoking, pet_friendly, taxes_included, cancellation, important_notes_one, important_notes_two, important_notes_three, important_notes_four, important_notes_five) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);', [checkInStart, checkInEnd, checkOut, kidFriendly, creditCards, ageRestriction, curfew, lockOut, nonSmoking, petFriendly, taxesIncluded, cancellation, importantNotesOne, importantNotesTwo, importantNotesThree, importantNotesFour, importantNotesFive]);
+    db.query('INSERT INTO rules (check_in_start, check_in_end, check_out, kid_friendly, credit_cards, age_restriction, curfew, lock_out, non_smoking, pet_friendly, taxes_included, cancellation, important_notes_one, important_notes_two, important_notes_three, important_notes_four, important_notes_five) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);', [checkInStart, checkInEnd, checkOut, kidFriendly, creditCards, ageRestriction, curfew, lockOut, nonSmoking, petFriendly, taxesIncluded, cancellation, importantNotesOne, importantNotesTwo, importantNotesThree, importantNotesFour, importantNotesFive], callback);
```

Comment:
You probably want to break up this line along with 71 & 83. They're a touch long. :)

Reply:
A good point to consider when deciding the configuration of the IDE/code editor. I keep forgetting that my code editor wraps lines.